src/Renderer/Label2DRenderer.js

```diff
@@ -186,13 +186,15 @@ class Label2DRenderer {
         if (!frustum.containsPoint(worldPosition.applyMatrix4(camera.matrixWorldInverse)) ||
         // Check if globe horizon culls the label
         // Do some horizon culling (if possible) if the tiles level is small enough.
-            label.horizonCullingPoint && GlobeLayer.horizonCulling(label.horizonCullingPoint) ||
-        // Check if content isn't present in visible labels
-            this.grid.visible.some((l) => {
-                // TODO for icon without text filter by position
-                const textContent = label.content.textContent;
-                return textContent !== '' && l.content.textContent.toLowerCase() == textContent.toLowerCase();
-            })) {
+            label.horizonCullingPoint && GlobeLayer.horizonCulling(label.horizonCullingPoint)
+            // Why do we might need this part ?
+            // || // Check if content isn't present in visible labels
```

Comment:
I think it was to avoid overlapping but I'm not sure we need it too so let's remove it completely in this case. We still have git if we need it later. @Desplandis WDYT?

Reply:
If we can't reproduce the overlapping and it seems to be the case here. I think we could totally remove this.

Reply:
As for now, it seems that it is not needed on what I tested and I asked @mgermerie why he used it in the first place.